### PR TITLE
test(Gmail Node): Add test coverage (no-changelog)

### PR DESCRIPTION
**Pull request by elsmr**
## Summary

image

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/NODE-2156/improve-gmail-node-testing

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/nodes-base/nodes/Google/Gmail/v2/loadOptions.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import type { ILoadOptionsFunctions, INodePropertyOptions } from 'n8n-workflow';
+
+import { googleApiRequest, googleApiRequestAllItems } from '../GenericFunctions';
+
+// Get all the labels to display them to user so that they can select them easily
```
Comment (tomi) [inline packages/nodes-base/nodes/Google/Gmail/v2/loadOptions.ts:5]: Can we use JSDoc so editors are able to show this when hovering

**Review by dana-gill — COMMENTED** [review]
I only reviewed until `7de92ddea0ced8fa64e3e765a83d958617fe5042` because tests were failing after that point, but from what I've seen, it looks good so far :) If you could fix the failing tests then ping me afterwards, that would be great 🙏

File: packages/nodes-base/nodes/Google/Gmail/v2/GmailV2.node.ts [hunk 2]
```diff
@@ -187,78 +181,9 @@ export class GmailV2 implements INodeType {
...
-						name: displayName,
-						value: alias.sendAsEmail,
-					});
-				}
-
-				return returnData;
-			},
+			getLabels,
+			getThreadMessages,
+			getGmailAliases,
```
Comment (dana-gill) [inline packages/nodes-base/nodes/Google/Gmail/v2/GmailV2.node.ts:186]: Also, this is great 🙌  It makes it much easier to read

**Comment by elsmr** [comment]
Tests were only failing on CI/CD, eventually figured out that it's because of a timezone difference 😅, fixed!

Could you do another review @dana-gill ?

**Review by dana-gill — APPROVED** [review]
lgtm 🥳

**Comment by janober** [comment]
Got released with [`n8n@1.73.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.73.0)

**Pull request merged**